eng/common/pipelines/templates/steps/verify-links.yml

```diff
@@ -1,19 +1,27 @@
 parameters:
   Directory: 'not-specified'
-  IgnoreLinksFile: "$(Build.SourcesDirectory)/eng/ignore-links.txt"
-  
+  IgnoreLinksFile: '$(Build.SourcesDirectory)/eng/ignore-links.txt'
+  WorkingDirectory: '$(System.DefaultWorkingDirectory)'
+  ScriptDirectory: 'eng/common/scripts'
+  Recursive: $false
+  Branchreplaceregex: '$(System.PullRequest.SourceRepositoryURI)'
+  Branchreplacementname: '$(System.PullRequest.SourceCommitId)'
+  Checklinkguidance: $true
+  Suffixes: '*.md' # The suffix regex. Support multiple suffixes, separated by comma. e.g '*.md, *.html'
```

Comment:
Why did switch away from producing the file list? By using a file list expression it gives the caller more flexibility, to include different file types as well as exclude files or directories, etc.

Reply:
We decided to put Urls as parameter in Verify-links.yml file. However, we are not able to take in the path parameters.
What we can put over the yml is:
```
Urls: 'Get-ChildItem -Path ./ -Recurse -Include *.md'
```
This is not flexible enough for changing the path. That's why android has to abandon the scripts.

Reply:
We cannot read any parameters in the above command, that's why it is not flexible. If this is acceptable for now. I can leave it as it is.

Reply:
I think that is acceptable. While we might not be able to access any of the variables here the person importing this template could use a common variable or just setup the path such that the relative "." path is at the correct place.